wrappers/node/test/suite1/ariesvcx-issuer-credential.test.ts

```diff
@@ -166,110 +167,47 @@ describe('IssuerCredential:', () => {
   })
 
   describe('sendCredential:', () => {
-    // todo : restore for aries
-    it.skip('success 1', async () => {
-      const connection = await connectionCreateConnect()
-      const issuerCredential = await issuerCredentialCreate()
-      await issuerCredential.sendOffer(connection)
-      VCXMock.setVcxMock(VCXMockMessage.IssuerCredentialReq)
-      VCXMock.setVcxMock(VCXMockMessage.UpdateIssuerCredential)
-      await issuerCredential.updateState()
-      assert.equal(await issuerCredential.getState(), StateType.RequestReceived)
-      await issuerCredential.sendCredential(connection)
-      assert.equal(await issuerCredential.getState(), StateType.Accepted)
-    })
-
-    it('success 2', async () => {
-      const connection = await connectionCreateConnect()
-      const issuerCredential = await issuerCredentialCreate()
-      const offer = await issuerCredential.getCredentialOfferMsg()
-      const cred = await Credential.create({ sourceId: 'name', offer, connection })
-      const pwDid = await connection.getPwDid()
-      const request = await cred.getRequestMessage({ myPwDid: pwDid, payment: 0 })
-      await issuerCredential.updateStateWithMessage(request)
-      assert.equal(await issuerCredential.getState(), StateType.RequestReceived)
-      const credMsg = await issuerCredential.getCredentialMsg(pwDid)
-      await cred.updateStateWithMessage(credMsg)
-      assert.equal(await cred.getState(), StateType.Accepted)
-    })
-
     it('throws: not initialized', async () => {
       const connection = await connectionCreateConnect()
       const issuerCredential = new IssuerCredential(null as any, {} as any)
       const error = await shouldThrow(() => issuerCredential.sendCredential(connection))
       assert.equal(error.vcxCode, VCXCode.INVALID_ISSUER_CREDENTIAL_HANDLE)
     })
 
-    it('throws: no offer', async () => {
+    // todo: recorder this test/behaviour in 4.0, issuerCredential is not throwing, only prints warning
+    it.skip('throws: no offer', async () => {
       const connection = await connectionCreateConnect()
       const issuerCredential = await issuerCredentialCreate()
       const error = await shouldThrow(() => issuerCredential.sendCredential(connection))
       assert.equal(error.vcxCode, VCXCode.NOT_READY)
     })
 
-    // todo : restore for aries
+    // todo: recorder this test/behaviour in 4.0, issuerCredential is not throwing, only prints warning
```

Comment:
yeah maybe, but it would need some more thinking around the state machine implementation, it seems they all never throw Errors, only transfer to finished state with problem report. I don't want to do these behaviour changes as part of this PR.